torch/autograd/variable.py

```diff
@@ -8,6 +8,15 @@
 import weakref
 from torch._six import imap
 from torch._C import _add_docstr
+import torch.onnx
+
+
+def _size_as_tensor(g, input):
+    return g.op('Shape', input)
+
+
+def _resize_from_tensor(g, input, shape):
```

Comment:
Wasn't the whole point of ONNX refactors to keep symbolica in a single place? Can we please remove them here? This is only polluting logic that shouldn't have anything to do with ONNX